skipruntime-ts/core/src/internals/skipruntime_impl.ts

```diff
@@ -264,21 +264,23 @@ export class ContextImpl extends SkFrozen implements Context {
   asyncLazy<
     K extends TJSON,
     V extends TJSON,
-    P extends TJSON,
+    AsyncParams extends TJSON,
     Params extends Param[],
```

Comment:
Just a note for ourselves: I think we still have some work to do to explain the difference between these two types of params clearly.